src/components/bs5/navbar/navbar.functions.js

```diff
@@ -67,10 +69,12 @@ export function initializeNavbar() {
     // Overlay show/hide events
     navbarCollapse.addEventListener('show.bs.collapse', function () {
       overlay.classList.add('show'); // Show the overlay
+      body.style.overflow = 'hidden'; // Prevent background scroll
     });
 
     navbarCollapse.addEventListener('hide.bs.collapse', function () {
       overlay.classList.remove('show'); // Hide the overlay
+      body.style.overflow = ''; // Reset body positioning
```

Comment:
@stvp-qld overflow:hidden as per health.qld.gov.au when mobile overlay is 'show' with states reset